src/main/java/org/docksidestage/app/web/profile/ProfileBean.java

```diff
@@ -0,0 +1,90 @@
+package org.docksidestage.app.web.profile;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import org.docksidestage.dbflute.exentity.Member;
+import org.docksidestage.dbflute.exentity.MemberService;
+import org.docksidestage.dbflute.exentity.Purchase;
+
+/**
+ * @author y.shimizu
+ */
+public class ProfileBean {
+
+    private String memberName;
+    private String memberStatusName;
+    private Integer servicePointCount;
+    private String serviceRankName;
+
+    private List<PurchasedProductBean> purchaseList;
+
+    public ProfileBean(Member member) {
+        this.memberName = member.getMemberName();
```

Comment:
just a suggestion: The mapping logic is deeply related to select logic (e.g. setupSelect).
So I prefer in constructor to in Action for visual check.